src/assets/i18n/en_US.json

```diff
@@ -240,8 +240,9 @@
   "account.order_template.move.added.text": "{{0}} has been added to <a callback=\"callbackHideDialogModal\" href=\"{{2}}\">{{1}}</a>.",
   "account.order_template.name.error.required": "If you want to create a new order template, please enter a name.",
   "account.order_template.new_from_basket.heading": "Create Order Template from Cart",
-  "account.order_template.new_from_basket_confirm.heading": "The order template \"{{0}}\" has been created and all items from the current cart have been transferred.",
+  "account.order_template.new_from_basket_confirm.heading": "The order template \"{{0}}\" has been created and all items have been transferred.",
   "account.order_template.new_from_order.button.create.label": "Create",
+  "account.order_template.new_from_order.heading": "Create Order Template from Order",
```

Comment:
Please use sentence case, also in line 247.

Reply:
We want keep the application consistent. That's why we should not use the sentence case at one place. Please, let's keep it here as it is and change the notation over the whole application at once.

Reply:
Arvid is already working on the task to unify to sentence case but I doubt that his file already contains these new keys. So I thought we'd better consider this for new keys already. Else it might be overlooked. If it doesn't take effect elsewhere, it's even more an argument for adjusting it while we spot it, isn't it? But you can also check with Arvid. I don't want to provoke any merge conflicts either.